src/server/status_server/region_meta.rs

```diff
@@ -0,0 +1,294 @@
+use engine_rocks::RocksEngine;
+use engine_traits::Iterable;
+use raft::{Progress, ProgressState, StateRole};
+use raftstore::store::Peer;
+use std::collections::HashMap;
+
+#[derive(Debug, Copy, Clone, Default, Serialize, Deserialize)]
+pub struct RegionSize {
+    pub default: Option<usize>,
+    pub write: Option<usize>,
+    pub lock: Option<usize>,
+}
+
+impl RegionSize {
+    pub fn scan(start_key: &[u8], end_key: &[u8], engine: &RocksEngine) -> Self {
+        let try_cf = |cf| {
+            let mut size = 0;
+            match engine.scan_cf(cf, start_key, end_key, false, |k, v| {
```

Comment:
This can be slow and use a lot of resource. I think it's OK to skip region size or use an approximate size.

Reply:
region size is removed